webpack.config.js

```diff
@@ -31,12 +21,60 @@ module.exports = {
         test: /\.(ts|js)x?$/,
         // exclude: /node_modules/,
         loader: "babel-loader",
+        options: { presets: ["@babel/env", "@babel/preset-react"] },
+      },
+      {
+        test: /\.css$/,
+        use: [MiniCssExtractPlugin.loader, "css-loader"],
       },
     ],
   },
-  plugins: [],
+  plugins: [new MiniCssExtractPlugin()],
   externals: {
     "canadv.ash": "commonjs canadv.ash",
     kolmafia: "commonjs kolmafia",
   },
 };
+
+const scriptsConfig = Object.assign(
+  {
+    entry: {
+      garbo: "./src/index.ts",
+      "garbo-combat": "./src/combat.ts",
+    },
+    output: {
+      path: path.resolve(__dirname, "KoLmafia", "scripts", "garbage-collector"),
+      filename: "[name].js",
+      libraryTarget: "commonjs",
+    },
+  },
+  sharedConfig
+);
+
+// handle the file creating the garbo UI html file
+const otherRelayConfig = Object.assign(
```

Comment:
I think you want webpack-merge here.

Reply:
I'm not familiar with webpack-merge, or webpack for that matter. I spent a little trying to get it to work, but wasn't able to wrap my ahead around what it should look like. If you could show me an example of someone using webpack-merge to handle multiple outputs, I'd be happy to use it.

Reply:
Turns out I had broken the webpack config doing something earlier. Using webpack-merge now.